helix-view/src/editor.rs

```diff
@@ -1069,6 +1069,14 @@ impl Editor {
         self.tree.transpose();
     }
 
+    pub fn resize_buffer(&mut self, resize_type: Resize, dimension: Dimension) {
+        self.tree.resize_buffer(resize_type, dimension);
+    }
+
+    pub fn buffer_expand_mode(&mut self) {
+        self.tree.buffer_expand_mode();
+    }
```

Comment:
I think the name and description for this command should say `toggle`. I'm also a little hesitant to call it a "mode" since that would skew the terminology for editor modes.

Maybe something like `toggle_focus_window`?